src/main/java/spireMapOverhaul/zones/gremlinTown/monsters/GremlinJerk.java

```diff
@@ -0,0 +1,112 @@
+package spireMapOverhaul.zones.gremlinTown.monsters;
+
+import basemod.abstracts.CustomMonster;
+import com.badlogic.gdx.math.MathUtils;
+import com.esotericsoftware.spine.AnimationState;
+import com.megacrit.cardcrawl.actions.animations.AnimateSlowAttackAction;
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.actions.common.RollMoveAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.localization.MonsterStrings;
+import spireMapOverhaul.SpireAnniversary6Mod;
+import spireMapOverhaul.zones.gremlinTown.powers.SapPower;
+import spireMapOverhaul.zones.invasion.powers.DrawReductionSingleTurnPower;
+
+import static spireMapOverhaul.util.Wiz.*;
+
+public class GremlinJerk extends CustomMonster
```

Comment:
Gremlin Jerk seems reasonable given the context of the encounter. It's basically Gremlin Assassin, but more HP/damage, but fought solo.